antora-ui-camel/src/css/frontpage.css

```
@@ -74,29 +74,30 @@ header.frontpage p {
   line-height: 1.7rem;
 }
 
-.frontpage a.button-dark,
-.frontpage a.button-dark:hover {
+a.button-dark,
+section.frontpage a.button-dark,
+section.frontpage a.button-dark:hover,
+a.button-dark:hover {
   white-space: nowrap;
   padding: 0.7rem 1rem;
   background-color: var(--color-camel-orange);
-  background-image: none;
-  background-repeat: no-repeat;
-  background-position: 0;
   color: white;
   font-weight: bold;
   margin-right: 0.5rem;
   border-radius: 3rem;
   outline: none;
   display: inline-block;
-  box-shadow: 0 4px #8e480b;
+  background-image: none;
 }
 
-.frontpage a.button-light {
+a.button-light,
+a.button-light:hover {
   padding: 0.7rem 1rem;
```

```suggestion
  padding: 0.3rem 1rem;
```

We could try a bit smaller buttons.